next.clayui.com/content/docs/components/drop-down.mdx

```diff
@@ -36,16 +39,81 @@ import {DropDown} from '../../../src/components/clay/DropDown';
+</div>
+
+DropDown is delivered for composition offering flexibility for customization. DropDown's composition gives you the space to create more complex DropDowns, such as adding a form within DropDown, and creating fast DropDowns with the high-level `ClayDropDownWithItems` component that covers most Lexicon specification use cases.
 
 <DropDown />
 
+## Composing
+
 You may want to use only the content of DropDown for other cases, such as mounting an `Autocomplete`, so use `DropDown.Menu` instead of `DropDown` which gives you the flexibility to only control content without a trigger.
 
 <div class="clay-site-alert alert alert-warning">
 	<code class="language-text">DropDown</code> consists of a <code class="language-text">DropDown.Menu</code> that controls the trigger and uses it as the basis for aligning DropDown content.
 </div>
 
+#### Controlling Trigger
+
+You may want to create a trigger that is not necessarily in the same tree as DropDown, due to HTML markup issues, for these cases you can use `<ClayDropDown.Menu />`.
+
+Using `<ClayDropDown.Menu />` allows you to better control the state of DropDown but you will have to deal with focus management, control expands and other details.
+
+```jsx
+import ClayDropDown from '@clayui/drop-down';
+import React, {useState, useRef} from 'react';
+
+const Menu = ({
+    children,
+    hasLeftSymbols,
+    hasRightSymbols,
+}) => {
+    const triggerElementRef = useRef(null);
+    const [expand, setExpand] = useState(false);
+    const menuElementRef = useRef(null);
+
+    const handleExpand = event => {
+        // This is not ideal for allowing you to have more than
+        // one trigger for the same content but it simulates the
+        // advantages of controlling `DropDown.Menu`.
+        triggerElementRef.current = event.target;
+
+        setExpand(!expand);
+    };
+
+    return (
+        <div {...}>
+            <div {...}>
+                <button type="button" onClick={handleExpand}>Home</button>
+            </div>
+            <div {...}>
+                <button type="button" onClick={handleExpand}>Product</button>
+            </div>
+
+            <ClayDropDown.Menu
+                active={expand}
+                alignElementRef={triggerElementRef}
+                hasLeftSymbols={hasLeftSymbols}
+                hasRightSymbols={hasRightSymbols}
+                onSetActive={() => setExpand(!expand)}
+                ref={menuElementRef}
+            >
+                {children}
+            </ClayDropDown.Menu>
+        </div>
+    );
+};
+```
+
+## DropDownWithItems
+
+Allows you to be able to create a faster DropDown, through its API you are able to create a Menu with groups of checkboxes and radios, links, buttons, search, caption... you will do the same as the components low-levels.
```

Comment:
```suggestion
Allows you to create a simple DropDown, through its API you are able to create a Menu with groups of checkboxes and radios, links, buttons, search, caption, etc.
```

Reply:
Maybe it would keep "would you do the same with low-level components" since it could do the same over there.

it's not fast 😱😞

Reply:
> you will do the same as the components low-levels

I'm not sure this last portion is helpful in clarifying what DropDownWithItems is. My goal would be to use text and descriptions like this to help users understand strange caveats, but for general purpose of a component I think pointing to examples and API is generally the best for communicating purpose. Its often hard to describe similarities or differences between high/low and much easier to just point to examples of seeing how its different.

Reply:
Hmm yeah, really better to point to an example.